docs/release-notes/november-2023.md

```diff
@@ -0,0 +1,138 @@
+  - [<i class="fas fa-external-link fa-xs"></i> 1.49.0](https://github.com/returntocorp/semgrep/releases/tag/v1.49.0)
+  - [<i class="fas fa-external-link fa-xs"></i> 1.50.0](https://github.com/returntocorp/semgrep/releases/tag/v1.50.0)
+  - [<i class="fas fa-external-link fa-xs"></i> 1.51.0](https://github.com/returntocorp/semgrep/releases/tag/v1.51.0)
+
+
+## 🌐 Semgrep Cloud Platform
+
+### Added
+
+- Semgrep now records the languages using interfile analysis during a scan. This enables the Semgrep team to measure new Pro Engine languages' performance impact and error rates. **For scans that don't send metrics, there is no change.** See [<i class="fas fa-external-link fa-xs"></i> Semgrep Privacy Policy](https://github.com/semgrep/semgrep/blob/develop/PRIVACY.md) for more information.
+- Added a link to the SSO documentation to help users set up SSO. <!-- 11485 -->
+- **CLI tool:** Added `--config code` and `--config secrets` flags to the `semgrep scan` command. When using these flags, the environment variable SEMGREP_REPO_NAME must be set. For example,
+    ```
+    $ SEMGREP_REPO_NAME=test_repo semgrep --config secrets
+    ```
+
+### Changed
+
+- Elixir language support now requires the Pro Engine. To scan Elixir codebases, enable the Pro Engine. <!-- 9308 -->
+- The Semgrep CLI tool now correctly counts the rules run on a codebase. Previously, Semgrep counted the total rules in the user's Policies or rulesets, including rules that did **not** have valid targets and therefore, did not actually run. <!-- 9130  -->
+- Updated instances of **returntocorp** to **Semgrep**. <!-- gh 112877 -->
+- **Semgrep Editor:** Rules created in the editor are private by default. This means only members of your organization can view rules you have created. To create a private rule visible only to you (an individual), ensure that you create the rule in your **individual account**. <!-- 11267 -->
+- `semgrep scan --config <product>` now uses the same endpoint as the `semgrep ci` to fetch the scan configuration. <!-- todo effect of this change -->
+- Improved error pages.
+
+
+### Fixed
+
+- **API:** Fixed an issue where the severities filter did not return the correct value. <!-- gh-11307 -->
+- **CLI tool:**
+    - The `--severity=[VALUE]` option, which can be added to a `semgrep scan` command, has been fixed. <!-- gh-9062 -->
+    - The `--sarif` flag no longer crashes when Semgrep itself encounters errors.
+- Semgrep now refuses to run incompatible versions of the Pro Engine, rather than crashing and returning a confusing error message. <!-- (gh-8873) -->
+- Fixed an issue where the CI provider icons disappeared from the **Scan new project in CI** window. The icons now appear. <!-- 11228 -->
+- Implemented minor fixes for the new onboarding flow. <!-- 11209, 11207 -->
+
+## 💻 Semgrep Code
+
+### Added
+
+- **Findings details page:** For users that have granted code access to Semgrep, you can now view **in-app code snippets** in Semgrep Cloud Platform.
```

Comment:
Did we GA this? @kmoses228 can you confirm?

Reply:
Removing!